addons/mail/static/src/composer/composer.scss

```diff
@@ -32,6 +32,14 @@
     }
 }
 
+.o-mail-Composer-actions button {
+    opacity: 75%;
+    &:hover {
+        background-color: rgba(0, 0, 0, 0.1);
+        opacity: 100%;
+    }
+}
```

Comment:
I think Stefano posted elsewhere that we should avoid customizing buttons (in favor of using standard, and improving standard when needed), and I agree.

Reply:
I could not find a general style of having button that has basic hover effect to darken around, and it works in both white and dark mode.